src/views/components/topology/chart/utils/nodeElement.js

```diff
@@ -49,7 +49,7 @@ export default (d3, graph, tool, funcs, tip) => {
     .attr('x', 2)
     .attr('y', 10)
     .attr('style', 'cursor: move;')
-    .attr('xlink:href', (d) => ((d.sla < 98 && d.isReal) ? icons.CUBEERROR : icons.CUBE));
+    .attr('xlink:href', (d) => (d.sla < 98 && d.isReal && d.cpm > 1 ? icons.CUBEERROR : icons.CUBE));
```

Comment:
Please check the condition of red cube @wu-sheng

Reply:
Let's reduce the expectation of sla to 95. And we need to describe this clearly on the UI.
- Healthy
- Unhealthy(Successful Rate < 95% and Traffic > 1 call/min)